backend/dataall/base/aws/sts.py

```diff
@@ -359,3 +359,13 @@ def generate_console_url(credentials, session_duration=None, region='eu-west-1',
 
         # Send final URL to stdout
         return request_url
+
+    @staticmethod
+    def is_assumable_pivot_role(accountid):
+        aws_session = SessionHelper.remote_session(accountid=accountid)
```

Comment:
will this ever return None or just raise an Exception if no session is created?

Reply:
It should just return False if no session is created

Reply:
What happens when the session goes into a ClientError exception (`dataall/base/aws/sts.py:64`?

Reply:
Oh. Okay I see where the issue might be present. I will catch the exception as well. Thanks for pointing that out @dlpzx , @noah-paige

Reply:
Looks good and clean now, resolving